packages/components/src/templates/next/components/complex/InfoCards/InfoCards.tsx

```diff
@@ -71,8 +73,14 @@ const InfoCardContainer = ({
 const InfoCardImage = ({
   imageUrl,
   imageAlt,
-}: Pick<SingleCardWithImageProps, "imageUrl" | "imageAlt">): JSX.Element => (
-  <div className={compoundStyles.cardImageContainer()}>
+  url,
+}: Pick<
+  SingleCardWithImageProps,
+  "imageUrl" | "imageAlt" | "url"
+>): JSX.Element => (
+  <div
+    className={compoundStyles.cardImageContainer({ isClickableCard: !!url })}
```

Comment:
@dcshzj I had to pass `icClickableCard` here as well because the hover effect applies to both `InfoCardImage` (hover on image) AND `cardTitle` (color change, arrow moves). Previously `group-hover` wasn't working on the `InfoCardImage`. Hope this is not weird in terms of practice though?

Reply:
Yup this works!